dnadoh/plotting_utils.py

```diff
@@ -0,0 +1,55 @@
+from scipy import stats
+import plotly.express as px
+import plotly.graph_objects as go
+
+
+def add_pvalue_annotation(df, fig, base, y_range, symbol=''):
+    """
+    arguments:
+    bases --- a list of two different days e.g. ['A','reference']
+    y_range --- a list of y_range in the form [y_min, y_max] in paper units
+    """
+    pvalue = stats.ttest_ind(
+        df[df['reference']=="reference"].weight,
+        df[df['reference']==base].weight)[1]
+    # print(pvalue)
+    if pvalue >= 0.05:
```

Comment:
i know 0.05 is the usual threshold, but i'd still create a named constant

Reply:
removed this file cause I don't even use it